src/main/resources/i18n/messages_de.properties

```diff
@@ -1,5 +1,6 @@
 # This file is automatically generated. Please do not edit this file. If you'd like to change the content please use crowdin.
 
+Audio.Like.MyAlbum=Meine Alben
```

Comment:
You'll need to add the term to the English file if you want it to be translatable on Crowdin